features/background-attachment.yml

```diff
@@ -0,0 +1,10 @@
+name: background-attachment
+description: The `background-attachment` CSS property sets whether the background layers of an element move as the element scrolls.
```

Comment:
I'm not super comfortable with "background layer" (I'm thinking of writing defensively due to `@layer`), so I'm going to try suggesting some changes here. Consider all of the "layer"-only suggestions optional—I'm happy to open a follow up PR if you prefer.

```suggestion
description: The `background-attachment` CSS property sets whether an element's background image or gradient moves as the element scrolls.
```

Reply:
This is a good point, and I don't think your rewrites introduced ambiguity, they look good to me!